input/pagecontent/hdecl-api.md

```diff
@@ -566,6 +566,25 @@ Tagastatakse päringu vastuses piiratud komplekt andmeid, profiilis määratakse
 ```
 _summary=true
 ```
+#### _profile
+Vastuses tagastatakse tervisedeklaratsioonid  vastavalt etteantud profiilile. 
+
+Praegu on toetatud järgmised profiilid:
+
+https://fhir.ee/hdecl/StructureDefinition/ee-health-declaration -  tervisedeklaratsioon tagastatakse täismahus
+
+https://fhir.ee/hdecl/StructureDefinition/ee-health-declaration-minimal - tagastatakse tervisedeklaratsiooni metaandmed koos kasutusalade ja nende kehtivustega. Kehtivused puuduvad in-progress olekus tervisedeklaratsiooni korral.
```

Comment:
Let's use link to profile resource here: `[EEHealthDeclarationMinimal](StructureDefinition-ee-health-declaration-minimal.html)`

Reply:
https://fhir.ee/hdecl/StructureDefinition/ee-health-declaration-minimal can be mentioned as well somehow.